### PR TITLE
feat: cancel recovery

### PR DESCRIPTION
**Pull request by yoanslvy**

### CODE REVIEW

File: src/services/delayModuleService.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+const DelayModule = new utils.Interface(delayModuleABI)
+
+export type DelayContext = {
+  delayModuleAddress: string
+  moduleFactoryAddress: string
+  recoveryCooldown: number
+  recoveryExpiration: number
+}
+
+export const getDelayAddress = async (
```
Comment (fdematos) [inline src/services/delayModuleService.ts:16]: The delay module address should be calculed and returned by the getWalletInfos method

File: src/utils/utils.ts [hunk 2]
```diff
@@ -1,5 +1,13 @@
 import { MetaTransaction } from 'ethers-multisend'
 
+import { DefaultSponsoredFunctions } from '../wallet/types'
+
+export const isDefaultSponsorisedFunction = (
```
Comment (fdematos) [inline src/utils/utils.ts:5]: Should had condition on to. addOwner, to should ne wallet. setNonce to should be delays contract

File: src/services/delayModuleService.ts (hunk 1 above)
Comment (yoanslvy) [reply src/services/delayModuleService.ts]: done

File: src/utils/utils.ts (hunk 2 above)
Comment (yoanslvy) [reply src/utils/utils.ts:5]: done

**Review by fdematos — APPROVED** [review]

**Review by fdematos — APPROVED** [review]

**Pull request merged**